addons/compat_spe/CfgAmmo/bullets.hpp

```diff
@@ -10,3 +10,28 @@ class SPE_B_127x99_Ball: SPE_Bullet_Vehicle_base {
 class SPE_B_127x99_API: SPE_B_127x99_Ball {
     EGVAR(vehicle_damage,incendiary) = 1;
 };
+
+// FLAK
+class SPE_Bullet_AA_base;
+class SPE_SprGr_FlaK_38: SPE_Bullet_AA_base
+{
```

Comment:
```suggestion
class SPE_SprGr_FlaK_38: SPE_Bullet_AA_base {
```
really minor style change